blog-service/2020/12-31.md

```diff
@@ -411,7 +411,7 @@ The key new components provided are:
 ---
 ## July 23, 2020 (Dashboard (New))
 
-New - We're proud to announce the release of [Dashboard (New)](/docs/dashboards-new), which provides you deeper visual control across logs and metrics data-sources, so you can build the perfect dashboard for your monitoring and troubleshooting needs. This is the first of many cool updates as we build towards a dashboard framework that is visually expressive, troubleshooting optimized, and hyper-performant.
+New - We're proud to announce the release of [Dashboard (New)](/docs/dashboards), which provides you deeper visual control across logs and metrics data-sources, so you can build the perfect dashboard for your monitoring and troubleshooting needs. This is the first of many cool updates as we build towards a dashboard framework that is visually expressive, troubleshooting optimized, and hyper-performant.
```

Comment:
please delete `(New)` here and in the heading as well

Reply:
updated.